apps/nextjs/src/app/home-page.tsx

```diff
@@ -268,6 +269,19 @@ export function HomePageContent({ pageData }: HomePageProps) {
               Find out more about Aila
             </BoldOakLink>
           </OakFlex>
+
+          <OakQuote
+            quote="Using AI to support my planning and teaching wasn’t something I’d really considered until I came across Aila. To say I was blown away would be an understatement!"
+            authorName="Avril"
+            authorTitle="Deputy Headteacher at Bedford Drive Primary School"
+          />
+          <OakQuote
+            quote="Incorporating Aila into my teaching toolkit has the potential to not only save me time - around 30 minutes per lesson - but also enhance the quality and effectiveness of my lessons, ultimately benefiting both myself and my students."
+            color="bg-decorative4-main"
+            authorName="James"
+            authorTitle="Teacher at St Cuthbert Mayne School"
```

Comment:
"St. Cuthbert"?

Reply:
Good catch, though it looks like it's how the school spells it https://www.st-cuthbertmayne.co.uk/